php.ini-development

```diff
@@ -1906,7 +1906,12 @@ ldap.max_links = -1
 ;opcache.file_cache_fallback=1
 
 ; Enables or disables copying of PHP code (text segment) into HUGE PAGES.
-; This should improve performance, but requires appropriate OS configuration.
+; Under certain circumstances (if only a single global PHP process is
+; started from which all others fork), this can increase performance
+; by about 1% because TLB misses are reduced.  On the other hand, this
```

Comment:
how did you come up with that number ?

Reply:
it might be more "circumstancial" at large, this is one case it might be higher/lower, the phrasing brings a bit too much "certainty" but that is just my take.

Reply:
Rephrased, changed "about 1%" to "a tiny amount" to take the certainty out of the explanation.